documentation/book/con-logging.adoc

```diff
@@ -0,0 +1,27 @@
+// This assembly is included in the following assemblies:
+//
+// assembly-deployment-configuration-kafka-mirror-maker.adoc
+
+// Save the context of the assembly that is including this one.
+// This is necessary for including assemblies in assemblies.
+// See also the complementary step on the last line of this file.
+
+[id='assembly-logging-{context}']
+= Logging
+
+Use the `logging.internal` and `logging.external` properties to configure loggers and logger levels.
```

Comment:
`logging.internal` and `logging.external` properties do not exist. It is always just `logging`.

```suggestion
Use the `logging` property to configure loggers and logger levels.
```